archive/app/views/notFound.scala.html

```diff
@@ -1,128 +0,0 @@
-                            <li class="zone-business"><a href="/business" class="zone-color">economy</a></li>
-                            <li class="zone-lifeandstyle"><a href="/lifeandstyle" class="zone-color">life</a></li>
-                            <li class="zone-lifeandstyle"><a href="/fashion" class="zone-color">fashion</a></li>
-                            <li class="zone-environment"><a href="/environment" class="zone-color">environment</a></li>
-                            <li><a href="/technology" class="zone-color">tech</a></li>
-                            <li><a href="/money" class="zone-color">money</a></li>
-                            <li><a href="/travel" class="zone-color">travel</a></li>
-                            <li><a href="https://soulmates.guardian.co.uk/" class="zone-color">soulmates</a></li>
-                            <li><a href="http://m.jobs.guardian.co.uk/" class="zone-color" target="_blank">jobs</a></li>
-                            <li><a href="/guardian-masterclasses" class="zone-color" target="_blank">masterclasses</a></li>
-                        </ul>
-                    </nav>
-                </div><!-- /.col-12 -->
-
-            </div><!-- /.row -->
-
-            <div class="footer zone-border zone-border-medium margin-top">
-                <ul class="inline with-separators">
-                    <li><a href="/help">Help</a></li>
-                    <li><a href="/help/contact-us">Contact us</a></li>
-                    <li><a target="_blank" href="https://www.surveymonkey.com/s/theguardian-beta-feedback">Feedback</a></li>
-                    <li><a href="/help/terms-of-service">Terms &amp; conditions</a></li>
-                    <li><a href="/help/privacy-policy">Privacy policy</a></li>
-                </ul>
-                <p><small>&copy; Guardian News and Media Limited or its affiliated companies. All rights reserved. Registered in England and Wales. No. 908396. Registered office: PO Box 68164, Kings Place, 90 York Way, London N1P 2AP</small></p>
-            </div>
-        </div><!-- /.fluid-wrap -->
-
-        <script>
-            (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-                    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-                m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-            })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
-            ga('create', 'UA-78705427-1', 'auto');
-            ga('set', 'dimension3', 'theguardian.com');
-            ga('set', 'dimension14', '404');
-            ga('send', 'pageview');
-        </script>
-
-        <script src="https://pasteup.guim.co.uk/js/lib/requirejs/2.1.5/require.min.js"></script>
-
-        <script>
-            require.config({
-                paths: {
-                    'ophan/http-status' : '//j.ophan.co.uk/ophan.http-status',
```

Comment:
@SiAdcock do you recognise this code? can we load this as a single file for now, or do we need to install/bundle ophan?